airflow/dag_processing/manager.py

```diff
@@ -653,6 +653,10 @@ def _refresh_dag_bundles(self):
         self.log.info("Refreshing DAG bundles")
 
         for bundle in self._dag_bundles:
+            # TODO: AIP-66 handle errors in the case of incomplete cloning? And test this.
+            #  What if the cloning/refreshing took too long(longer than the dag processor timeout)
+            if not bundle.is_initialized:
+                bundle.initialize()
```

Comment:
@jedcunningham , I noticed that if ssh connection is not configured well, then it will get stuck cloning the repo until the dag processor times out. On the restart, the dag processor will stay on crash loop. 
Dag processor default timeout is 50 secs.

Reply:
Hmm, that 50s timeout shouldn't apply here... But we should probably have some protection here. We can harden this later 👍 .